src/__tests__/http-test.js

```diff
@@ -9,20 +9,22 @@
  */
 
 // 80+ char lines are useful in describe/it, so ignore in this file.
+// Also ignore no-unused-expressions rule as chai 'expect' sintax violates it.
```

Comment:
@santino I think @wincent is referring to the typo - `sintax` should be `syntax` no?